Makefile

```diff
@@ -3,8 +3,38 @@ export BASE_BRANCH
 
 ifneq (,$(DAPPER_HOST_ARCH))
 
+CONTROLLER_GEN := $(CURDIR)/bin/controller-gen
+
 # Running in Dapper
 
+GO ?= go
+GOARCH = $(shell $(GO) env GOARCH)
+GOEXE = $(shell $(GO) env GOEXE)
+GOOS = $(shell $(GO) env GOOS)
+
+# Get the currently used golang install path (in GOPATH/bin, unless GOBIN is set)
+ifeq (,$(shell $(GO) env GOBIN))
+GOBIN=$(shell $(GO) env GOPATH)/bin
+else
+GOBIN=$(shell $(GO) env GOBIN)
+endif
```

Comment:
None of these used explicitly in the makefile.

Do we need them for actually running the generator or could it run without this?